test/json/v1beta1_api_resource_list.json

```diff
@@ -0,0 +1,11 @@
+{
+  "kind": "APIResourceList",
+  "groupVersion": "extensions/v1beta1",
```

Comment:
@abonas sorry about that. I checked the 1.7 api and deployments is still in `extensions\v1beta1`. it didn't get promoted.